packages/components/src/components/Menu/Menu.tsx

```diff
@@ -55,9 +55,6 @@ export const Menu = ({
         middleware: [offset(8), flip(), shift({ padding: 8 })],
         placement,
     });
-    useEffect(() => {
-        update();
```

Comment:
By removing the useEffect is it necessary to define update here?
const { refs, floatingStyles, update } = useFloating({...

Reply:
true, i removed it now 🛠️